### DISCUSSION TITLE
!important as a modifier, e.g. important:bg-blue-600

### DISCUSSION THREAD

**Discussion by dorianmariecom** [tailwindlabs/tailwindcss#5997] [Ideas]
?

I define some styles in CSS and need to overwrite it with tailwind so an important modifier would be nice, especially with the JIT

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

This already exists as using `!`. e.g.: `!text-center` or `hovert:!text-center`
More info: https://tailwindcss.com/docs/just-in-time-mode#built-in-important-modifier

**Reply by dorianmariecom** [reply]
oh nice thanks couldn't find it, it's perfect
